tfw

```diff
@@ -59,9 +55,27 @@ main() {
   : "${VARLIBDIR:=/var/lib}"
   : "${VARTMPDIR:=/var/tmp}"
 
+  export DEVDIR DIST ETCDIR OPTDIR RUNDIR TMPDIR USRBINDIR USRSBINDIR \
+    VARCACHEDIR VARLIBDIR VARTMPDIR
+
+  for rcfile in "${ETCDIR}/tfwrc" "${HOME}/.tfwrc"; do
+    if [[ -s "${rcfile}" ]]; then
+      # shellcheck source=/dev/null
+      source "${rcfile}" || __error "failed to source '${rcfile}'"
+    fi
+  done
+
+  for key in "${!__TFW_ALIASES__[@]}"; do
+    eval "__run_${key}() { __run_${__TFW_ALIASES__[${key}]} \"\${@}\"; }"
+  done
+
   local command="${1}"
   shift || true
-  if "__run_${command:-help}" "${@}"; then
+  if command -v "__run_${command:-help}" &>/dev/null &&
+    "__run_${command:-help}" "${@}"; then
+    exit 0
+  elif command -v "tfw-${command}" &>/dev/null &&
+    "tfw-${command}" "${@}"; then
```

Comment:
I feel like I've run into problems before using hyphens in commands, but I can't reproduce now, so oh well! :woman_shrugging:

Reply:
Also, maybe I'm reading this wrong, but aren't we losing error output if the command fails here? Or are we trusting the command to report?

Reply:
Sounds fun! :smile_cat:

Reply:
We shouldn't be losing error output for the command execution, which happens on line 78.  Line 77 is asking a question where the exit code is all that matters to me.  Does that help answer your question?